tidy3d/components/geometry.py

```diff
@@ -930,10 +944,30 @@ class PolySlab(Planar):
         units=MICROMETER,
     )
 
+    dilation: float = pydantic.Field(
+        0.0,
+        title="Dilation",
+        description="Dilation of the polygon in the base by shifting each edge along its "
+        "normal outwardws direction by a distance; a negative value corresponds to erosion.",
+        units=MICROMETER,
+    )
+
+    sidewall_angle: float = pydantic.Field(
+        np.pi / 2,
+        title="Sidewall angle",
+        description="Angle between the sidewall and the base of the polyslab. "
+        "``sidewall_angle=np.pi/2`` (default) specifies vertical wall, "
```

Comment:
might be too late to change and not a big deal, but is it standard for the angle to be defined w.r.t the substrate / base instead of the vertical sidewall?

Reply:
based on this convention: https://support.lumerical.com/hc/en-us/articles/360034382394-Layer-builder-Simulation-object